packages/open-payments/src/client/incoming-payment.ts

```diff
@@ -9,28 +9,25 @@ import {
 } from '../types'
 import { get, post } from './requests'
 
-interface GetArgs {
+interface RequestWithUrlArgs {
   url: string
   accessToken: string
 }
 
-interface PostArgs<T = undefined> {
-  url: string
-  body?: T
-  accessToken: string
-}
-
-interface ListGetArgs {
+interface RequestWithPaymentPointerArgs {
```

Comment:
What do you think of `(Sub)ResourceRequestArgs` and `CollectionRequestArgs`?

That might also somewhat future-proof this for
- https://github.com/interledger/open-payments/issues/206

Reply:
Yup, I like that better